helm/charts/determined/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
 name: determined
 description: A Helm chart for Determined
```

Comment:
Change description to:
`An open source deep learning training platform with support for distributed training and hyperparameter search.`